src/libsync/propagatedownload.cpp

```diff
@@ -828,7 +829,11 @@ void PropagateDownloadFile::slotGetFinished()
                 &propagator()->_anotherSyncNeeded, errorBody);
         }
 
-        done(status, errorString);
+        if (err == QNetworkReply::RemoteHostClosedError || err == QNetworkReply::TemporaryNetworkFailureError) {
+            errorCategory = ErrorCategory::NetworkError;
```

Comment:
@matthieugallien You have added a method `ErrorCategory PropagatorJob::errorCategoryFromNetworkError(QNetworkReply::NetworkError error)`. Why not use it here?